Fix compilation with Pyhton 3.6 and kernel 5.4

jopereira opened the pull request:
This fixes various broken imports, the syntax of an except clause, and an issue with recent kernels.

jopereira committed:
Fix various missing __init__.py and broken imports

jopereira committed:
Fix compatibility with kernel 5.4

fntneves commented:
Please, remove the `__pycache__` dir and add it to the `.gitignore`.

Does it still work with Python 2?
I'm afraid we need to update the README.

fntneves reviewed (changes requested):
Remove the `falcon/util/__pycache__/net.cpython-39.pyc` and add it to the `.gitignore`, please.

jopereira committed:
Update README.md

This inclides detailed commands for Ubuntu 18.04 LTS (Bionic)
with kernel 5.4.0 and Python 3.6 as a reference platform.

jopereira committed:
Fix Kafka and JSON writers for Python3
